src/main/java/com/exactpro/th2/conn/MessageSender.java

```diff
@@ -122,4 +125,12 @@ private Event createErrorEvent(String eventType) {
                 .type(eventType);
     }
 
+    private IMetadata toSailfishMetadata(RawMessage protoMsg) {
+        if (!protoMsg.hasParentEventId()) {
+            return IMetadata.EMPTY;
```

Comment:
Maybe add conn's root event id to report about custom sending

Reply:
You mean if we received a message to send without parent ID we will attach the event about sending this message to our root event?

Reply:
I mean we can attach the message to a sub-event of the conn's root.